config/config.exs

```diff
@@ -45,6 +45,15 @@ config :plenario, PlenarioEtl.Scheduler,
     {"* * * * *", {PlenarioEtl.ScheduledJobs, :refresh_datasets, []}}
   ]
 
+config :plenario, PlenarioAot.AotScheduler,
+  global: true,
+  jobs: [
+    {"*/5 * * * *", {PlenarioAot.AotScheduler, :import_aot_data, []}}
```

Comment:
We know that AoT data is refreshed every 5 minutes, so we strip away the refresh cadence metadata and just set the scheduler to run the job every 5 minutes.